ft_volumenormalise.m

```diff
@@ -388,13 +414,10 @@
 ft_postamble debug
 ft_postamble trackconfig
 
-% Remember the initial and normalisation parameters in the configuration, this allows
-% redoing the transformations without any computations (e.g. estimating them on a T1
-% and applying them on a T2)
-%
-% they are added only here to prevent them to be removed when doing the trackconfig
-cfg.initial   = initial;
+% remember the normalisation parameters in the configuration
+% maintain this order for the time being to prevent them to be removed when doing the trackconfig
 cfg.spmparams = params;
+cfg.final     = final;
```

Comment:
`cfg.final` (which is a 4x4 affine transformation, even if a nonlinear transformation was applied) was removed in Sept 2021 in favor of `cfg.initial` (which is an affine transformation) plus `cfg.spmparams` (which can be affine or nonlinear). See https://github.com/fieldtrip/fieldtrip/pull/1875

Reply:
This should also be fixed. It seems that my changes were not written in the most recent version of ft_volumenormalise.